api/lib/controllers/participant.js

```diff
@@ -18,4 +20,38 @@ module.exports = {
       res.status( 500 ).send( error );
     }
   },
+  getMeetings: async( req, res ) => {
+    logger.debug( "finding meetings " + JSON.stringify( req.params ) );
+
+    const conn = mongoose.connection;
+    const col = conn.collection( "meetings" );
+
+    const { email } = req.body;
+    const participantMeetings = new Array;
+
+    try{
+      const participants = await Participant.find({ email });
+
+      let meetingIds = new Array();
+
+      participants.forEach( ( participant ) => {
+        meetingIds.push( participant.meeting_id );
+      });
+
+      const query = { _id: { $in: meetingIds } };
+
+      const cursorMeetings = await col.find( query, {}).toArray();
+
+      logger.debug(
+        "participant meetings found " + JSON.stringify( cursorMeetings )
+      );
+      res.status( 201 ).send( cursorMeetings );
+    }
+    catch( error ) {
+      res.status( 500 ).send( error );
+    }
+    finally {
+      await conn.disconnect();
```

Comment:
Since `try`, `catch`, and `finally` are connected blocks, you generally format it like this:
```js
try {
  throw new Error('im not sure what you were expecting to happen');
} catch( err ) {
  console.error( err.message );
} finally {
  console.log('this is very important so always do it');
}
```
It is similar to `if`/`else` blocks.